js/game.js

```diff
@@ -43,6 +44,7 @@ var game = {
     playerDice: [],
     remainingRerolls: 0,
     selectedDiceKeys: new Set(),
+    playerStartsBets: true,
```

Comment:
what about something more obvious about this being a boolean, eg. isPlayerStarting?